src/mosquitto_broker_internal.h

```diff
@@ -507,6 +507,7 @@ struct mosquitto__bridge_topic{
 	char *remote_topic; /* topic prefixed with remote_prefix */
 	enum mosquitto__bridge_direction direction;
 	uint8_t qos;
+	struct mosquitto__bridge_topic *next;
```

Comment:
Could you please put this earlier in the struct so there's no chance of any struct packing problems in the future?